server/test/utils.ts

```diff
@@ -60,11 +49,12 @@ async function createUser({
     github_username: github_username ?? randomEmptyChance(20, randUserName()),
     website: website ?? randomEmptyChance(20, randUrl()),
   };
+}
 
+async function createUser(options: Partial<UserObject> = {}) {
```

Comment:
i think we need more distinctive names. what's the difference between `makeUser` and `createUser`?  "make" and "create" are synonyms so for someone unfamiliar with the codebase, it may be hard to understand when to use each function.

Reply:
That one was on me, something I stole from Laravel. lol
[make](https://laravel.com/api/10.x/Illuminate/Database/Eloquent/Builder.html#method_make) makes an unsaved model while [create](https://laravel.com/api/10.x/Illuminate/Database/Eloquent/Builder.html#method_create) creates a saved model.

I agree, the names aren't very intuitive but for some reason I've never gotten them confused. 😂

Reply:
yeah I also think changing may be beneficial?